java/Ice/context/client/build.gradle.kts

```diff
@@ -7,12 +7,9 @@ dependencies {
 
 sourceSets {
     main {
-        // Add the Greeter.ice file from the parent slice directory the main source set.
+        // Add the Greeter.ice file from the parent slice directory to the main source set.
         slice {
             srcDirs("../slice")
-            // By default a Slice source set includes all Slice files in the srcDirs directories
-            // Here we override the default behavior by specifying the list of Slice files to include.
-            setIncludes(listOf("Greeter.ice"))
```

Comment:
This is required in Ice/greeter because we have both Greeter.ice and GreeterAMD.ice and only want to include one. But not required here or in the new secure demo.